src/assets/nameCards.json

```diff
@@ -0,0 +1,23 @@
+{
+  "data": [
+    {
+      "id": 1,
+      "firstName": "Sweet",
+      "lastName": "Booboo",
+      "nickName": "Boo",
+      "avatar": null,
+      "title": "Software Engineer",
+      "company": "Google",
+      "city": "San Francisco",
+      "state": "CA",
+      "skills": ["css", "html", "JavaScript"],
+      "interests": "cats",
+      "socialMedias": {
+        "LinkedIn": "https://www.linkedin.com",
```

Comment:
@BayLadyCoder Should we use camel case here as well?

Reply:
Yes, I agree. @KateChantha can update later.